src/storagesync/azext_storagesync/custom.py

```diff
@@ -0,0 +1,201 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+# pylint: disable=line-too-long
+# pylint: disable=too-many-statements
+# pylint: disable=too-many-lines
+# pylint: disable=too-many-locals
+# pylint: disable=unused-argument
+from azure.cli.core.util import sdk_no_wait
+
+
+def create_storagesync_storage_sync_service(client,
+                                            resource_group_name,
+                                            storage_sync_service_name,
+                                            tags=None,
+                                            location=None):
+    body = {}
+    body['location'] = location  # str
+    body['tags'] = tags  # dictionary
+    return client.create(resource_group_name=resource_group_name, storage_sync_service_name=storage_sync_service_name, parameters=body)
+
+
+def delete_storagesync_storage_sync_service(client,
+                                            resource_group_name,
+                                            storage_sync_service_name):
+    return client.delete(resource_group_name=resource_group_name, storage_sync_service_name=storage_sync_service_name)
+
+
+def get_storagesync_storage_sync_service(client,
+                                         resource_group_name,
+                                         storage_sync_service_name):
+    return client.get(resource_group_name=resource_group_name, storage_sync_service_name=storage_sync_service_name)
+
+
+def list_storagesync_storage_sync_service(client,
+                                          resource_group_name=None):
+    if resource_group_name is not None:
```

Comment:
Use `if not resource_group_name:`  in order to use the workaround `-g=` to temparorily disable appending default resource group as we discussed.

Reply:
will update it.

---
In reply to: [392833381](https://github.com/Azure/azure-cli-extensions/pull/1389#discussion_r392833381) [](ancestors = 392833381)